src/test/java/com/exasol/exaoperation/plugin/PluginIT.java

```diff
@@ -95,4 +108,24 @@ void testRunPluginScriptCatchesUnsupportedOperationException()
         assertRunPlugInScriptCatchesException(new UnsupportedOperationException());
     }
 
+    @Test
+    void testListPlugins() {
+        List<String> plugins = CONTAINER.getExaOperation().getPluginNames();
+        assertThat( plugins.size(), equalTo( 1 ) );
```

Comment:
If you use multiple asserts in one test case, please wrap them in `assertAll(...)` and give each one an explanatory description.